### DISCUSSION TITLE
Jamulus OS

### DISCUSSION THREAD

**Discussion by olenz927** [jamulussoftware/jamulus#1388] [Technical Support & Help] [answered]
Hello!
Has anybody experience with Jamulus OS? I installed it in the latest version on a USB Stick. It runs well. When I want to use the integrated install procedure to bring Jamulus OS to the HDD it seems to work fine, but I can't login with the user I have to provide with the installation.
I would like to know if this is a general issue and how I can fix it.

thanks a lot ... Oliver

**Comment by gilgongo** [comment, marked as answer]
You may need to contact the developer of Jamulus OS directly. https://sourceforge.net/projects/jamulus-os/support  It's not maintained here.
